full/src/main/java/apoc/custom/Signatures.java

```diff
@@ -126,23 +127,51 @@ private DefaultParameterValue defaultValue(SignatureParser.DefaultValueContext d
             return DefaultParameterValue.nullValue(type);
         if (v.boolValue() != null)
             return DefaultParameterValue.ntBoolean(Boolean.parseBoolean(v.boolValue().getText()));
-        if (v.stringValue() != null)
-            return DefaultParameterValue.ntString(v.stringValue().getText());
-        if (v.INT_VALUE() != null)
-            return DefaultParameterValue.ntInteger(Integer.parseInt(v.INT_VALUE().getText()));
-        if (v.FLOAT_VALUE() != null)
-            return DefaultParameterValue.ntFloat(Float.parseFloat(v.FLOAT_VALUE().getText()));
+        final SignatureParser.StringValueContext stringCxt = v.stringValue();
+        if (stringCxt != null) {
+            
+            String text = stringCxt.getText();
+            if (stringCxt.SINGLE_QUOTED_STRING_VALUE() != null || stringCxt.QUOTED_STRING_VALUE() != null) {
+                text = text.substring(1, text.length() - 1);
+            }
+            return DefaultParameterValue.ntString(text);
+        }
+        if (v.INT_VALUE() != null) {
+            final String text = v.INT_VALUE().getText();
+            return getDefaultParameterValue(type, text, () -> DefaultParameterValue.ntInteger(Integer.parseInt(text)));
```

Comment:
Btw. we should fix this as it should be Long.parseLong()

Reply:
Changed